resources/files/windows/run_facter_interactive.bat

```diff
@@ -2,4 +2,4 @@
 @echo off
 SETLOCAL
 call "%~dp0environment.bat" %0 %*
-@elevate.exe "%~dp0facter_interactive.bat"
+"%PUPPET_DIR%\bin\elevate.exe" "%~dp0facter_interactive.bat"

```

Comment:
We should keep the `@` I think, see: https://stackoverflow.com/a/21077142/6648276

same for `run_puppet_interactive.bat`